test/lib/linter/rules/snapshots/XMLInJs.ts.md

```diff
@@ -0,0 +1,220 @@
+# Snapshot report for `test/lib/linter/rules/XMLInJs.ts`
+
+The actual snapshot is saved in `XMLInJs.ts.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## General: FragmentLoad.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 1,
+        fatalErrorCount: 0,
+        filePath: 'FragmentLoad.js',
+        messages: [
+          {
+            column: 12,
+            line: 7,
+            message: 'Use of deprecated property \'tap\' of class \'Button\'',
+            messageDetails: 'Deprecated test message',
+            ruleId: 'no-deprecated-api',
+            severity: 2,
+          },
+        ],
+        warningCount: 0,
+      },
+    ]
+
+## General: SapUIFragment.js
```

Comment:
Same hint here and for some other files regarding the naming convention.

Reply:
Done